tests/helpers/database.py

```diff
@@ -0,0 +1,7 @@
+from sqlalchemy.ext.automap import automap_base
+
+
+def table_class_for(table, engine):
```

Comment:
what does this function do? table class? but it returns a list of classes, not? Consider renaming it ;-)